Set/SetCard.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  SetCard.swift
+//  Set
+//
+//  Created by Alon Reik on 15/03/2021.
+//
+
+import Foundation
+
+// it is a class because i want it to be a reference type
+class SetCard: Equatable
+{
+    
+    /* ------------
+     Static Methods
+     ------------- */
+
+    /*
+        Returns true iff both SetCard instances has equal values in every property. 
```

Comment:
iff = if and only if. 
meaning: If both setCard instances has equal value in every property, the function returns true. Otherwise, returns false.
(It is my bad, I forgot to document my intentions in that regard).